### ISSUE TITLE
Send an email (V2) cannot save attachment content

### ISSUE THREAD

**Issue by huidongl** [Azure/LogicAppsUX#4086]
### Describe the Bug with repro steps

The office 365 Outlook "Send an email (v2)" action, after adding the attachment name and content in the UI, in the code view, the attachment content become empty. 

We tried to use dynamic content and hard coded value, both failed to be added to the code. 

### What type of Logic App Is this happening in?

Standard (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Workflow JSON

```json
{
  "definition": {
    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
    "actions": {
      "Read_blob_content": {
        "type": "ServiceProvider",
        "inputs": {
          "parameters": {
            "containerName": "destination",
            "blobName": "File1.txt"
          },
          "serviceProviderConfiguration": {
            "connectionName": "AzureBlob-7",
            "operationId": "readBlob",
            "serviceProviderId": "/serviceProviders/AzureBlob"
          }
        },
        "runAfter": {}
      },
      "Send_an_email_(V2)": {
        "type": "ApiConnection",
        "inputs": {
          "host": {
            "connection": {
              "referenceName": "office365"
            }
          },
          "method": "post",
          "body": {
            "To": "huidongliu@microsoft.com",
            "Subject": "2401290030003259 test",
            "Body": "<p>test attachment</p>",
            "Attachments": [
              {
                "Name": "attachment1.txt",
                "ContentBytes": ""
              },
              {
                "Name": "attachment2.txt",
                "ContentBytes": ""
              }
            ]
          },
          "path": "/v2/Mail"
        },
        "runAfter": {
          "Compose": [
            "SUCCEEDED"
          ]
        }
      },
      "Compose": {
        "type": "Compose",
        "inputs": "@body('Read_blob_content')?['content']",
        "runAfter": {
          "Read_blob_content": [
            "SUCCEEDED"
          ]
        }
      }
    },
    "contentVersion": "1.0.0.0",
    "outputs": {},
    "triggers": {
      "Recurrence": {
        "type": "Recurrence",
        "recurrence": {
          "frequency": "Month",
          "interval": 15,
          "startTime": "2023-03-05 05:41:50Z",
          "timeZone": "China Standard Time"
        }
      }
    }
  },
  "connectionReferences": {
    "AzureBlob-7": {
      "connection": {
        "id": "/serviceProviders/AzureBlob/connections/AzureBlob-7"
      },
      "connectionName": "hlwork3",
      "api": {
        "id": "/serviceProviders/AzureBlob"
      }
    },
    "office365": {
      "connection": {
        "id": "/subscriptions/cc0ecbfb-0faf-492d-8996-be21d0865ca6/resourceGroups/Blob403/providers/Microsoft.Web/connections/office365"
      },
      "connectionName": "office365",
      "api": {
        "id": "/subscriptions/cc0ecbfb-0faf-492d-8996-be21d0865ca6/providers/Microsoft.Web/locations/southeastasia/managedApis/office365"
      },
      "authentication": {
        "type": "ManagedServiceIdentity"
      }
    }
  },
  "parameters": {
    "we": {
      "type": "String",
      "value": "{\"p\":\"c\"}"
    }
  }
}
```

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/92560328/e795b714-bc4c-4164-9a37-9c36e62f2325)

![image](https://github.com/Azure/LogicAppsUX/assets/92560328/8ef2fe99-cc16-4602-8dfd-5945e1508a75)

### Browser

Chrome

**Comment by Eric-B-Wu** [comment]
Hi @huidongl thanks for reporting this issue. I believe I should have fixed it in #4007 and will be fixed in our next release, however, let me confirm

**Comment by Eric-B-Wu** [comment]
closing issue as release completed in all regions
